.teamcity/src/main/kotlin/model/bucket-extensions.kt

```diff
@@ -48,6 +48,11 @@ fun <T, R> splitIntoBuckets(
 
     val expectedBucketSize = list.sumOf(toIntFunction) / expectedBucketNumber
 
+    if (expectedBucketSize == 0) {
```

Comment:
❓ When does this happen?

Reply:
In [`AllVersionsIntegMultiVersion` build](https://builds.gradle.org/project/Gradle_Master_Check_AllVersionsIntegMultiVersion_33?branch=%3Cdefault%3E&buildTypeTab=overview&mode=builds#all-projects), where only a few subprojects have test executions.